components/modelmeshserving/modelmeshserving.go

```diff
@@ -126,6 +128,13 @@ func (m *ModelMeshServing) ReconcileComponent(ctx context.Context,
 		}
 	}
 
+	extraParamsMap := map[string]string{
+		"nim-state": getNimManagementFlag(owner),
+	}
```

Comment:
so when modelmesh is removed, wont be a set in modelmesh's parames.env for nim-state value ?

Reply:
are we planning to dump a default nim-state: remove in modelmesh params.env?

Reply:
IMO I would say No... We only care about nim state when Kserve is enabled